src/squirrel/supertux_api.cpp

```diff
@@ -787,6 +787,38 @@ static void resume_target_timer()
   GameSession::current()->set_target_timer_paused(false);
 }
 
+/**
+ * @scripting
+ * @description Override the Item Pocket setting in the Level.
+ * @param string allow Can be "on", "off", or "inherit" (use the setting in the Level)
+ */
+static void override_allow_item_pocket(const std::string& allow)
```

Comment:
The C++ function name is different. Make sure they both stay the same for accuracy in documentation.

```suggestion
static void override_item_pocket(const std::string& allow)
```